connectors/http/http-base/src/main/java/io/camunda/connector/http/base/model/HttpCommonResult.java

```diff
@@ -16,14 +16,16 @@
  */
 package io.camunda.connector.http.base.model;
 
-import com.google.common.base.Objects;
+import com.fasterxml.jackson.annotation.JsonIgnore;
 import java.util.Map;
+import java.util.Objects;
 
 public class HttpCommonResult {
```

Comment:
Can also potentially be converted into a record

Reply:
Done!